scripts/check-sample-lnd-conf.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# This script performs different checks on the sample-lnd.conf file:
+# 1. Checks that all relevant options of lnd are included
+# 2. Verifies that defaults are labeled if there are also further examples
+# 3. Checks that all default values of lnd are mentioned correctly, including
+#    empty defaults and booleans which are set to false by default.
+
+set -e
+
+TAGS="$1"
+CONF_FILE=${2:-sample-lnd.conf}
+
+# We are reading the default values of lnd from lnd --help. To avoid formatting
+# issues the width of the terminal is set to 240. This needs a workaround for
+# CI where we don't have an interactive terminal.
+if [ -t 0 ]; then
+    stty cols 240
+    LND_HELP="$(go run -tags="$TAGS" \
+        github.com/lightningnetwork/lnd/cmd/lnd --help)"" --end"
+else
+    tmux new-session -d -s simulated-terminal -x 240 -y 9999
+    tmux send-keys -t simulated-terminal.0 "go run -tags=\"$TAGS\" \
+        github.com/lightningnetwork/lnd/cmd/lnd --help; tmux wait -S run" ENTER
+    tmux wait-for run
+    LND_HELP="$(tmux capture-pane -t simulated-terminal.0 -p)"" --end"
+    tmux kill-session -t simulated-terminal
+fi
+
+# LND_OPTIONS is a list of all options of lnd including the equal sign, which
+# is needed to distinguish between booleans and other variables.
+# It is created by reading the first two columns of lnd --help. 
+
```

Comment:
nit: remove extra newline between comment and code if comment is meant for the code following it immediately (here and further below).

Reply:
removed twice